app/config/AppConfig.scala

```diff
@@ -64,6 +64,8 @@ class AppConfig @Inject()(
 
   lazy val customsDeclareExportsBaseUrl = servicesConfig.baseUrl("customs-declare-exports")
 
+  lazy val getRedirectionLink: String = s"${servicesConfig.baseUrl("customs-email-frontend")}/manage-email-cds/service/customs-declare-exports"
```

Comment:
Could you follow the pattern to put the path into config?

Reply:
Yeah, I've just realised that this way won't work as the 'RedirectionLink' is a public DNS link and I'd have to add an entry for each environment to correctly set the host. Will have to update SFUS similarly!